tests/acceptance/docker/src/ocis-base.yml

```diff
@@ -23,6 +27,8 @@ services:
     volumes:
       - ../../../config:/drone/src/tests/config
       - oCISownCloud10testsuite:/srv
+    entrypoint: /bin/sh
```

Comment:
My tests failed because my `ocis_server` didn't start
`/bin/sh: ocis: not found`

does it works for you?

Reply:
yes, it works for me. How did you run the test?

Reply:
@SwikritiT could use full path in the `command` if that's the case. `/usr/bin/ocis`

Reply:
@ScharfViktor can you try again with my latest commit and see it it works, I'll address your other reviews in meantime

Reply:
same, with `/bin/sh: /usr/bin/ocis: not found` I'm going to google how to solve this problem.

Reply:
@ScharfViktor what system are you using?

Reply:
I could not up `ocis_server` on my mac. skip please it to don't lock your PR. I try to solve it in separate issue later